framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -366,6 +368,17 @@ public AnnotatedTypeFactory(BaseTypeChecker checker) {
         }
     }
 
+    /**
+     * Issue an error if any of the support qualifiers has @Target that contains something besides
```

Comment:
If the annotation doesn't contain TYPE_USE or TYPE_PARAMETER but does contain something else, I don't think this method throws an error, but the documentation suggests that it does.  (And it probably should.  You could just check the return value of hasTypeQualifierElementTypes.)
Also, replace "issue" by "throw", since issuing an error sounds like using the report mechanism.